docs/src/content/docs/snippets/wallet-get-info.mdx

```diff
@@ -11,7 +11,7 @@ import Js from '/src/partials/js-prerequisites.mdx'
 
 For a client to request a grant from an authorization server, the client must first verify the validity of the wallet address and get the URL of the wallet's authorization server.
 
-These code snippets enable an authenticated client to verify a wallet address, get the basic information required to construct a new transaction, and discover the auth server’s URL. Note that unauthenticated clients can also call the Open Payments API to get wallet address information.
+These code snippets enable an authenticated client to verify a wallet address, get the basic information required to construct a new transaction, and discover the auth server’s URL. Note that unauthenticated clients can also call the Open Payments APIs to get wallet address information.
```

Comment:
In this instance, unauthenticated clients can only call the Wallet Address Server API, not the Resource Server or Auth Server APIs. The plural should be changed back to singular and we need to figure out how to actually refer to these APIs in the docs.

Reply:
I'm going to switch back to singular and refer to it as the "Wallet Address API". That's what it's called in the OpenAPI spec: https://github.com/interledger/open-payments/blob/main/openapi/wallet-address-server.yaml